electron/app/components/ViewBar/ViewStage/ViewStageParameter.tsx

```diff
@@ -130,11 +130,10 @@ const convert = (value, placeholder) => {
   return value;
 };
 
-const makePlaceholder = (parameter, type, defaultValue) =>
-  [
-    defaultValue ? [parameter, defaultValue].join("=") : parameter,
-    toTypeAnnotation(type),
-  ].join(": ");
+const makePlaceholder = ({ placeholder, parameter }) => {
+  if (placeholder !== undefined) return placeholder;
+  return parameter;
+};
```

Comment:
My error was here. If a placeholder is not defined, the parameter name is used instead.

Reply:
Yup, I've seen this before :)
`typeof placeholder !== "undefined"` also would have worked, although I prefer the one you went with.